components/script/dom/globalscope.rs

```diff
@@ -124,6 +141,7 @@ impl GlobalScope {
         timer_event_chan: IpcSender<TimerEvent>,
         origin: MutableOrigin,
         microtask_queue: Rc<MicrotaskQueue>,
+        list_auto_close_worker: DomRefCell<Vec<AutoCloseWorker>>,
```

Comment:
We don't need to pass this in as an argument. We can simply initialize it with a default value instead.